itunesSearch/itunesSearch/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SearchResultsTableViewController.swift
+//  itunesSearch
+//
+//  Created by Bradley Yin on 8/6/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController {
+    
+    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet weak var segmentControl: UISegmentedControl!
+    
+    
+    let searchResultController = SearchResultController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        searchBar.delegate = self
+
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return searchResultController.searchResults.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "ResultCell", for: indexPath)
+        cell.textLabel?.text = searchResultController.searchResults[indexPath.row].title
+        cell.detailTextLabel?.text = searchResultController.searchResults[indexPath.row].creator
+        return cell
+    }
+
+}
+
+extension SearchResultsTableViewController: UISearchBarDelegate{
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text else{return}
+        var resultType: ResultType!
+        
+        switch segmentControl.selectedSegmentIndex {
+        case 1:
+            resultType = .musicTrack
+        case 2:
+            resultType = .movie
+        default:
+            resultType = .software
+        }
+        
+        searchResultController.performSearch(searchTerm: searchTerm, resultType: resultType) { (error) in
+            if let error = error{
+                print("error searching: \(error)")
+                return
+            }
+            DispatchQueue.main.async {
+                self.tableView.reloadData()
+            }
+        }
+        
```

Comment:
I don't think other people care as much as I do, but I don't like the empty space right here!